public/index.html

```diff
@@ -14,6 +14,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
     <title>Gróa</title>
+    <link rel="stylesheet" href="./reset.css" />
```

Comment:
I also moved reset CSS file, as I kept receiving unexpected rem sizing for fonts when it was in the index.js.